### PR TITLE
Add alerting policy guide

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: content/en/docs/project-user-guide/alerting/alerting-policy.md [hunk 1]
```diff
@@ -6,68 +6,59 @@ linkTitle: "Alerting Policy (Workload Level)"
...
 
-Log in to the console as `project-regular` and go to your project. Navigate to **Alerting Policy** under **Monitoring & Alerting**, then click **Create**.
+2. In the dialog that appears, provide the basic information as follows. Click **Next** to continue.
 
-![alerting_policy_workload_level_create](/images/docs/alerting/alerting_policy_workload_level_create.png)
+   - **Name**. A concise and clear name as its unique identifier, such as `alert-demo`.
+   - **Alias**. Help you distinguish alerting policies better.
+   - **Description**. A brief introduction to the alerting policy.
+   - **Duration (Minutes)**. An alert will be firing when the conditions defined for an alerting policy are met at any give point in the time range.
+   - **Type**. The alert type. Allowed values include **Warning**, **Major** and **Critical**.
```
[the remark this answers is not in the document]
Comment (Sherlock113) [reply content/en/docs/project-user-guide/alerting/alerting-policy.md]: Changed in https://github.com/kubesphere/console/pull/1838.

[the remark this answers is not in the document]
Comment (junotx) [reply content/en/docs/project-user-guide/alerting/alerting-policy.md]: verified

**Review by junotx — APPROVED** [review]
/lgtm

**Pull request merged**